backend/lib/azimutt_web/admin/project/_projects_table.html.heex

```diff
@@ -0,0 +1,34 @@
+<div class="flex flex-col border-l border-1 border-slate-800 pl-4 pb-4">
+    <div class="text-2xl font-bold text-slate-100 py-2"> <%= assigns.projects |> Enum.count() %> projects</div>
+    <div class="min-w-full overflow-hidden overflow-x-auto align-middle shadow sm:rounded-sm">
+        <table class="w-full">
+            <thead class="text-gray-100 bg-slate-800 text-xs font-semibold">
+            <tr>
+                <th class="px-1 py-1 text-center" scope="col">Created</th>
+                <th class="px-1 py-1 text-center" scope="col">Name</th>
+                <th class="px-1 py-1 text-center" scope="col">T/C/L</th>
```

Comment:
some span with title would be helpful to know what are T/C/L ;)
Maybe notes are more important than columns (and later memos), so having a T/L/N

Reply:
Rhoo :D ok je te rajoute ca ;) 
pour info j'ai rajouté les sources depuis